tracing/README.md

```diff
@@ -15,19 +15,18 @@ package main
 
 import (
   "go.uber.org/fx/tracing"
-  "go.uber.org/fx/ulog"
 
+	"github.com/uber-go/tally"
```

Comment:
nit: indents look off (on github). may be right if lint passed, but it's displaying weirdly for me